distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -559,22 +559,20 @@ CDDL-1.1 -- licenses/LICENSE-CDDL-1.1.txt
     - org.glassfish.hk2-osgi-resource-locator-1.0.3.jar
     - org.glassfish.hk2.external-aopalliance-repackaged-2.6.1.jar
  * Jersey
-    - org.glassfish.jersey.containers-jersey-container-servlet-2.31.jar
-    - org.glassfish.jersey.containers-jersey-container-servlet-core-2.31.jar
-    - org.glassfish.jersey.core-jersey-client-2.31.jar
-    - org.glassfish.jersey.core-jersey-common-2.31.jar
-    - org.glassfish.jersey.core-jersey-server-2.31.jar
-    - org.glassfish.jersey.ext-jersey-entity-filtering-2.31.jar
-    - org.glassfish.jersey.media-jersey-media-jaxb-2.31.jar
-    - org.glassfish.jersey.media-jersey-media-json-jackson-2.31.jar
-    - org.glassfish.jersey.media-jersey-media-multipart-2.31.jar
-    - org.glassfish.jersey.inject-jersey-hk2-2.31.jar
+    - org.glassfish.jersey.containers-jersey-container-servlet-2.34.jar
```

Comment:
The log title should probably be changed into "Updating to Jersey 2.34"

Reply:
Updated the PR title